tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "fastify-tsconfig",
+  "compilerOptions": {
+    "outDir": "dist",
+    "typeRoots": ["@types", "node_modules/@types"]
+  },
+  "include": ["@types", "src/**/*.ts"]
```

Comment:
Now we have editor error messages in `test` because it is not included anymore.

But it works.

Not sure how to fix this properly.

Reply:
You mean just use:

```
  "include": [
    "src/**/*.ts",
    "test/**/*.ts"
  ],
  ```

Reply:
Ho, yep for @types.
But if I include `"test/**/*.ts` it will compile test in dist locally, increasing build time.
How could I include without compiling?